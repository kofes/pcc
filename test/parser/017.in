var
ident1, ident2 : integer;
var
