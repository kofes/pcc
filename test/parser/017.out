Unexpected end of file in pos (4, 1); expected '`identifier`';
